holidays/countries/russia.py

```diff
@@ -23,6 +23,11 @@ class Russia(HolidayBase):
     country = "RU"
     default_language = "ru"
 
+    special_holidays = {
+        2023: ((FEB, 24, "День защитника Отечества"),), # Bridge day for 1/Jan, src: https://www.consultant.ru/document/cons_doc_LAW_425407/
+        2023: ((MAY, 8, "День Победы"),), # Bridge day for 8/Jan, src: https://www.consultant.ru/document/cons_doc_LAW_425407/
```

Comment:
These keys are identical and the latter key overwrites former one.
This is why we emphasize on importance of [adding tests](https://github.com/dr-prodigy/python-holidays/blob/beta/tests/countries/test_russia.py).